File: Programming with C#/3.Software Technologies-March2017/7.PHP-MVC-SymfonyOverview/Calculator/src/CalculatorBundle/Entity/Calculator.php
<?php
namespace CalculatorBundle\Entity;
class Calculator
{
    /*
     * @var float
     */
    private $leftOperand;
    /*
     * @var float
     */
    private $rightOperand;
     /*
     * @var string
     */
    private $operator;

    /*
     * Set left operand
     * @param float $operator
     * @return Calculator
     */
    public function getLeftOperand()
    {
    return $this->leftOperand;

    }
    public function setLeftOperand($operand)
    {
        $this->leftOperand = $operand;
        return $this;
    }
    /*
     * Set right operand
     * @param float $operator
     * @return Calculator
     */
    public function getRightOperand()
    {
        return $this->rightOperand;

    }
    public function setRightOperand($operand)
    {
        $this->rightOperand = $operand;
        return $this;
    }
    /*
     * Set right operand
     * @param string $operator
     * @return Calculator
     */
    public function getOperator()
    {
        return $this->operator;

    }
    public function setOperator($operator)
    {
        $this->operator = $operator;
        return $this;
    }
}

?>

File: Programming with C#/3.Software Technologies-March2017/13.Java Syntax, Basic Web/Calculator/src/main/java/com/softuni/models/Calculator.java
package com.softuni.models;


public class Calculator {
    private double leftOperand;
    private String operator;
    private double rightOperand;

    public Calculator(double leftOperand, String operator, double rightOperand) {
        this.leftOperand = leftOperand;
        this.operator = operator;
        this.rightOperand = rightOperand;
    }

    public double getLeftOperand() {
        return leftOperand;
    }

    public void setLeftOperand(double leftOperand) {
        this.leftOperand = leftOperand;
    }

    public String getOperator() {
        return operator;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public double getRightOperand() {
        return rightOperand;
    }

    public void setRightOperand(double rightOperand) {
        this.rightOperand = rightOperand;
    }

    public double calculateResult() {
        double result;

        switch(operator) {
            case "+": result = this.getLeftOperand() + this.getRightOperand();
                break;
            case "-": result = this.getLeftOperand() - this.getRightOperand();
                break;
            case "*": result = this.getLeftOperand() * this.getRightOperand();
                break;
            case "/": result = this.getLeftOperand() / this.getRightOperand();
                break;
            default: result = 0;
            break;

        }

        return result;
    }


}
